h2o-core/src/main/java/water/api/ModelMetricsHandler.java

```diff
@@ -368,6 +368,8 @@ else if (s.deep_features_hidden_layer_name != null){
           predictions = new Frame(Key.<Frame>make(parms._predictions_name), predictions.names(), predictions.vecs());
           DKV.put(predictions._key, predictions);
         }
+          j.setWarnings(parms._model.get_warnings());
```

Comment:
This will remove all prior warnings that were logged on the job. It would probably be safer to append to the existing set of warnings.